connect/mirror/README.md

```diff
@@ -0,0 +1,187 @@
+
+# MirrorMaker 2.0
```

Comment:
The readme could be re-worded a bit to start with a quickstart simplest case and then build from there to increasingly more complex cases. This could be done at a later pass too.